google_apis.md

```diff
@@ -7,17 +7,30 @@ and use that project for all your environments.
 
 ![google_app_1](images/google_app_1.png)
 
-## Enable Google Maps JS APIs
+* After creating it, make sure that it is selected in the header and enable the Google APIs that you need for this project under `APIs & Services > Library`
 
-* When enabling the Google Maps Javascript API key you can generate one or more keys for your environments.
-What is important, is to restrict the usage of the key to certain domains.
+* When asked, attach the `wg-education CC` as billing account.
 
-![google_app_2](images/google_app_2.png)
+## API key generation
 
-Remember to enable the specific domain for production and both the renuoapp.ch and herokuapp.com domains.
+**Maps JavaScript API**
 
-* When asked, attach the `wg-education CC` as billing account.
+In order to user the Google APIs you will need to generate new credentials. Because we are using only one project per application, but would like to separate the usage in development from the one in production, name the keys like so: `MASTER Maps Public Key` and `DEVELOP Maps Public Key`.
```

Comment:
The restrictions speak against that. If we want to add HTTP Referrer restrictions to the key, you run into problems while geocoding with the same key, because they are not supported. This was the reason Simon and I came up with the idea of using two separate keys (`public` and `private`)

Reply:
My "what about using only one key" was meant in regard to the environments. so, using one key for testing, develop, and master. sorry for the confusion. I agree with you regarding the usage of different keys for different purposes.